bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -135,20 +135,22 @@ export default {
           allPoints = allEquipmentPoints(equipment)
           points = [
             ...findPoints(allPoints, 'Point_Status', false),
-            ...findPoints(allPoints, 'Point_Control', true)
+            ...findPoints(allPoints, 'Point_Control', true),
+            ...findPoints(allPoints, 'Point_Switch', false)
           ]
           if (points.length) return points
           return equipment.filter((e) => e.points.length === 0).map((e) => e.item)
         case 'screens':
           equipment = [
-            ...findEquipment(this.element.equipment, 'Equipment_AudioVisual_Display', true),
+            ...findEquipment(this.element.equipment, 'Equipment_AudioVisual_Display_Television', true),
```

Comment:
I am in doubt on this. What is the recommendation for the use of the Screen tag?

Reply:
I dunno. Someone in the review process insisted that a screen is something different than a display. And honestly I just rolled over and didn't fight it..

Reply:
So I include Television and Screen under screens. And Projector under projectors.

Reply:
Or we could double count Projector both under screens and under projectors?